worlds/pokemon_rb/docs/setup_fr.md

```diff
@@ -0,0 +1,102 @@
+- Le client Archipelago intégré, qui peut être installé [ici](https://github.com/ArchipelagoMW/Archipelago/releases)
+   (sélectionnez `Pokemon Client` lors de l'installation).
+- Fichiers ROM Pokémon Rouge et/ou Bleu. La communauté Archipelago ne peut pas les fournir.
+
+## Logiciel en option
+
+-[Pokémon Red and Blue Archipelago Map Tracker](https://github.com/j-imbo/pkmnrb_jim/releases/latest), à utiliser avec [PopTracker](https://github.com/black-sliver/PopTracker/releases)
+
+
+## Configuration de BizHawk
+
+Une fois BizHawk installé, ouvrez EmuHawk et modifiez les paramètres suivants :
+
+- (≤ 2,8) Allez dans Config > Personnaliser. Passez à l'onglet Avancé, puis faites passer le Lua Core de "NLua+KopiLua" à "Lua+LuaInterface". 
+Ensuite redémarrez EmuHawk. Ceci est nécessaire pour que le script Lua fonctionne correctement.
+   **REMARQUE : Même si « Lua+LuaInterface » est déjà sélectionné, basculez entre les deux options et resélectionnez-la. Nouvelles installations**
+   **des versions plus récentes d'EmuHawk ont tendance à afficher "Lua+LuaInterface" comme option sélectionnée par défaut mais se chargent toujours**
+   ** "NLua+KopiLua" jusqu'à ce que cette étape soit terminée.**
+- Sous Config > Personnaliser > Avancé, assurez-vous que la case AutoSaveRAM est cochée et cliquez sur le bouton 5s.
+   Cela réduit la possibilité de perdre des données de sauvegarde en cas de panne de l'émulateur.
+- Sous Config > Personnaliser, cochez la case "Exécuter en arrière-plan". Cela empêchera la déconnexion du client pendant EmuHawk s'exécute en arrière-plan.
+
+Il est fortement recommandé d'associer des extensions de rom GB (\*.gb) à l'EmuHawk que nous venons d'installer.
+Pour ce faire, nous devons simplement rechercher n'importe quelle rom Gameboy que nous possédons, faire un clic droit et sélectionner "Ouvrir avec...", déplier
+la liste qui apparaît et sélectionnez l'option du bas "Rechercher une autre application", puis accédez au dossier BizHawk et sélectionnez EmuHawk.exe.
+
+## Configurer votre fichier YAML
+
+### Qu'est-ce qu'un fichier YAML et pourquoi en ai-je besoin ?
+
+Votre fichier YAML contient un ensemble d'options de configuration qui fournissent au générateur des informations sur la manière dont il doit
+générer votre jeu. Chaque joueur d'un multimonde fournira son propre fichier YAML. Cette configuration permet à chaque joueur de profiter
+une expérience personnalisée à leur goût, et les différents joueurs du même multimonde peuvent tous avoir des options différentes.
+
+### Où puis-je obtenir un fichier YAML ?
+
+Vous pouvez générer un yaml ou télécharger un modèle en visitant la [Page des paramètres du joueur Pokémon Rouge et Bleu](/games/Pokemon%20Red%20and%20Blue/player-settings)
+
+Il est important de noter que l'option `game_version` détermine le fichier ROM qui sera corrigé.
```

Comment:
Later, "patch" is mentionned in "pokémon patch file". I'm not sure of the word "corrigé" here.